auth/store.go

```diff
@@ -207,17 +222,26 @@ func (as *authStore) AuthDisable() {
 	tx.Unlock()
 	b.ForceCommit()
 
-	as.enabledMu.Lock()
 	as.enabled = false
-	as.enabledMu.Unlock()
 
 	as.simpleTokensMu.Lock()
 	as.simpleTokens = make(map[string]string) // invalidate all tokens
+	as.simpleTokenKeeper.stop()
 	as.simpleTokensMu.Unlock()
 
 	plog.Noticef("Authentication disabled")
 }
 
+func (as *authStore) Close() error {
```

Comment:
I don't think it matters. Once it's closed, the authStore object should be discarded, so the tokens are thrown away too.